spec/plaindatetime.html

```diff
@@ -1092,20 +1074,7 @@ <h1>ToTemporalDateTimeFields ( _temporalDateTimeLike_, _fieldNames_ )</h1>
       <emu-note>The value of ? ToIntegerOrInfinity(*undefined*) is 0.</emu-note>
       <emu-alg>
         1. Assert: Type(_temporalDateTimeLike_) is Object.
```

Comment:
This assertion is also at the start of PrepareTemporalFields, so can be removed. (Also applies below)